spaces/models.py

```diff
@@ -3,7 +3,45 @@
 
 
 class District(models.Model):
+    TYPE_MAJOR_DISTRICT = "major_district"
+    TYPE_DISTRICT = "district"
+    TYPE_SUB_DISTRICT = "sub_district"
+
+    TYPE_CHOICES = (
+        (TYPE_MAJOR_DISTRICT, _("Major district")),
+        (TYPE_DISTRICT, _("District")),
```

Comment:
So i assume these are pretty much set in stone? Like that there only exists district, neighborhood etc? So there is no need to make this super flexible and just use one space/place model, and with attributes in the model control whether it's a district, neighborhood or rentable space or whatever so we could have pretty much infinite hierarchy?

Reply:
Districts, neighborhood, real estate and building are models for mostly search, filter and information display purposes. Major districts, Districts and Subdistricts can be used to filter reservation units according to Jira documents. 

[Subdivisions_of_Helsinki](https://en.wikipedia.org/wiki/Subdivisions_of_Helsinki). According to this wiki page neighborhoods and districts are different system. They usually overlap but sometimes don't.

**I guess neighborhood could also be one type of district model. I will change it so.**

Space is only entity here that can be attached to reservation unit. Space can be a whole building, room of building, part of room of building, outside area etc.

Reply:
If you want to further discuss this we can take slack call or google meet. Its pretty hard to talk about this thoroughly here.

Reply:
No need to, this is enough information 👍